src/main/java/gov/usgs/cida/pubs/dao/BaseDao.java

```diff
@@ -31,9 +31,9 @@
 	public static final String ID_SEARCH = "id";
 	public static final String TEXT_SEARCH = "text";
 
-	public static final String PAGE_ROW_START = "page_row_start";
-	public static final String PAGE_SIZE = "page_size";
-	public static final String PAGE_NUMBER = "page_number";
+	public static final String PAGE_ROW_START = "pageRowStart";
```

Comment:
Won't this change the pw-publication-mvc-service query parameters? If so it will break the UI

Reply:
Since this is in the DAO layer, this shouldn't affect the service parameter names.  I think DS is just unifying the parameter names from the service layer to the DAO layer.

But, as Ed pointed out, Ed did change these param names in the service layer to camel case.  Ed is separately working on a UI change to match those new names.

Reply:
I talked to Dave before he left and this does need to be fixed. That's why I reassigned his ticket to him and put it back into In Progress.